apps/webapp/src/app/(Magic-Link)/magic-link/page.tsx

```
@@ -0,0 +1,614 @@
+          ))}
+        </div>
+      </div>
+
+      {/* Basic Auth Dialog */}
+      <Dialog open={openBasicAuthDialog} onOpenChange={onCloseBasicAuthDialog}>
+        <DialogContent className="bg-white text-black rounded-lg shadow-lg max-w-md w-full p-0 overflow-hidden">
+          <div className="flex justify-between items-center p-4">
+            <button
+              onClick={() => onCloseBasicAuthDialog(false)}
+              className="text-gray-500 hover:text-gray-700"
+            >
```

Biome

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)